src/model.jl

```diff
@@ -6,62 +6,197 @@ export create_model, solve_model
 Create the model using the `graph` structure, the parameters and sets.
 """
 
-function create_model(graph, params, sets; verbose = false)
+function create_model(graph, params, sets; verbose = false, write_lp_file = false)
     # Sets unpacking
-    A = sets.assets
-    Ac = sets.assets_consumer
-    # Ap = sets.assets_producer
-    Ai = sets.assets_investment
-    F = [(A[e.src], A[e.dst]) for e in edges(graph)]
-    K = sets.time_steps
-    RP = sets.rep_periods
+    A   = sets.assets
+    Ac  = sets.assets_consumer
+    Ap  = sets.assets_producer
+    Ai  = sets.assets_investment
+    As  = sets.assets_storage
+    Ah  = sets.assets_hub
+    Acv = sets.assets_conversion
+    F   = [(A[e.src], A[e.dst]) for e ∈ edges(graph)] # f[1] -> source, f[2] -> destination
+    Fi  = [f for f ∈ F if params.flows_investable[f]]
+    Ft  = [f for f ∈ F if params.flows_is_transport[f]]
+    K   = sets.time_steps
+    RP  = sets.rep_periods
 
     # Model
     model = Model(HiGHS.Optimizer)
     set_attribute(model, "output_flag", verbose)
 
     # Variables
-    @variable(model, 0 ≤ v_flow[F, RP, K])         #flow from asset a to asset aa [MW]
-    @variable(model, 0 ≤ v_investment[Ai], Int)  #number of installed asset units [N]
+    @variable(model, v_flow[F, RP, K])         #flow from asset a to asset aa [MW]
```

Comment:
Maybe now change `v_flow` too since the prefix has been changed for everything else?

Reply:
I agree, I will change it to flow